src/modules/user/repository/user.repository.ts

```diff
@@ -89,7 +89,7 @@ export class UserRepository extends Repository<UsersEntity> {
 
     user.recoverPasswordToken = recoverPasswordToken;
 
-    await this.update(id, user).catch(handleError);
+    await this.save(user);
```

Comment:
await this.update(id, user).catch(handleError);

deixar da maneira como estava antes

Reply:
Ok, atualizei para voltar ao .update